chgnet/model/dynamics.py

```diff
@@ -651,6 +653,29 @@ def set_atoms(self, atoms: Atoms):
         self.dyn.atoms = atoms
         self.dyn.atoms.calc = calculator
 
+    def upper_triangular_cell(self):
+        """Transform to upper triangular cell."""
```

Comment:
For posterity's sake, can you add a comment to this doc string explaining what this is useful for.